dll/win32/browseui/precomp.h

```diff
@@ -79,4 +79,13 @@ struct ShellSettings
     BOOL Load();
 };
 
+struct CabinetStateSettings : CABINETSTATE
+{
+    BOOL fFullPathAddress = TRUE;
+
+    BOOL Load();
+};
```

Comment:
At some point it could be cleaner to have all those settings structures definitions in their own header file.

Reply:
Added a header file for settings.